proto/sources.proto

```diff
@@ -338,6 +338,12 @@ message Huggingface {
   bool include_prs = 19;
 }
 
+enum JiraInstallationType {
+  JIRA_INSTALLATION_TYPE_AUTODETECT = 0;
+  JIRA_INSTALLATION_TYPE_CLOUD = 1;
+  JIRA_INSTALLATION_TYPE_DATA_CENTER = 2;
+}
```

Comment:
It's a good practice ([mentioned in the protobuf guide](https://protobuf.dev/programming-guides/proto3/#enum:~:text=%7D-,Enum%20Default%20Value,-The%20default%20value)) to have the first value of the enum be `ENUM_TYPE_NAME_UNSPECIFIED` or `ENUM_TYPE_NAME_UNKNOWN`

Reply:
Hmm, it's a matter of what we've been doing and what's the right way of doing it....... I'm not sure about this.

Personally I agree with the guide. Having the default value of the enum be an `UNKNOWN` value makes sense because then your downstream application can know that the caller did not explicitly specify a value. In our case, we would probably map this `UNKNOWN` to `AUTODETECT`, but still there will be a way to differentiate if the user actually wanted `AUTODETECT` or if they just didn't specify a value.